### ISSUE TITLE
Farhan: Talk with the team about the layout of the website

### ISSUE THREAD

**Issue by ImaCoderLol** [CSAllenISD/2023-ISP-TherapySession#9]

**Comment by HyunseoLee0** [comment]
We have to talk about the layout of the website. For example, we should talk like how the website is structured, which button should be placed in main page, etc.

**Comment by ImaCoderLol** [comment]
yeah we could make like a sort of draft layout for buttons, links, text, etc. before making the real thing
